_data/events/Oaxaca.json

```diff
@@ -0,0 +1,17 @@
+{
+  "title": "Global Day of Code Retreat 2017 Oaxaca, México",
+  "url": "https://www.eventbrite.com/e/global-day-of-coderetreat-oaxaca-2017-tickets-39236483378",
+  "moderators": [
+    "@oaxacarb"
```

Comment:
Please add the actual moderators here if possible ☺️👌

Reply:
Updated information of moderators 👍